x-pack/metricbeat/module/googlecloud/compute/_meta/data.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_index": "metricbeat-8.0.0-2019.11.27-000001",
+    "_id": "H4iRrm4B7vvv-udOwwFO",
+    "_version": 1,
+    "_score": null,
+    "_source": {
+        "@timestamp": "2019-11-27T20:30:00.000Z",
+        "ecs": {
+            "version": "1.2.0"
+        },
+        "host": {
+            "name": "mcastro"
+        },
+        "agent": {
+            "version": "8.0.0",
+            "type": "metricbeat",
+            "ephemeral_id": "cb56f25c-fc0e-4bbc-9c49-4551c1a97ab5",
+            "hostname": "mcastro",
+            "id": "7e36a073-1a32-4a94-b65b-4c7f971fb228"
+        },
+        "cloud": {
+            "availability_zone": "us-central1-a",
+            "account": {
+                "id": "elastic-metricbeat"
+            },
+            "provider": "googlecloud",
+            "instance": {
+                "id": "3296438752514919533",
+                "name": "instance-3"
+            }
+        },
+        "event": {
+            "module": "googlecloud",
+            "duration": 1252249339,
+            "dataset": "googlecloud.compute"
+        },
+        "metricset": {
+            "period": 300000,
+            "name": "compute"
+        },
+        "service": {
+            "type": "googlecloud"
+        },
+        "googlecloud": {
+            "compute": {
+                "labels": {
+                    "user": {
+                        "label_key": "label_value"
+                    },
+                    "metadata": {
+                        "metadata_key": "metadata_value"
+                    }
+                },
+                "instance": {
+                    "uptime": 60,
+                    "cpu": {
+                        "usage_time": 0.049102112533603304,
+                        "utilization": 0.004462245110474328
+                    }
+                }
+            }
+        }
+    },
+    "fields": {
+        "@timestamp": [
+            "2019-11-27T20:30:00.000Z"
+        ]
+    },
+    "highlight": {
```

Comment:
Hmm sorry I probably missed something, whats this `hightlight.metricset.name`?

Reply:
Good question but I have no clue :sweat_smile: I think it's something that Kibana has attached to the event (I copy pasted the json from kibana). I'll investigate it